.github/workflows/tests.yml

```
@@ -50,8 +50,10 @@ jobs:
 
           # Test more available versions of CPython on Linux.
           - os: "ubuntu-latest"
-            python-version: "3.6"
-          - os: "ubuntu-latest"
+            # v3.6 (exclusively referenced) is not supported by GitHub anymore)
+            # v3.6.8 is fixed since it is the version CentOS/Rocky/RedHat v8 uses
+            python-version: "3.6.8"
+          - os: "ubuntu-20.04"
             python-version: "3.7"
```

This one belongs to Python 3.7. Should work with `latest`.

```suggestion
          - os: "ubuntu-latest"
            python-version: "3.7"
```

---

Ah, got it, i'll revert